content/en/actions/datastore/use.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Use Datastores with Apps and Workflows
+disable_toc: false
+further_reading:
+- link: "service_management/app_builder/build"
+  tag: "Documentation"
+  text: "Build Apps"
+- link: "service_management/workflows/build"
+  tag: "Documentation"
+  text: "Build Workflows"
+- link: "https://www.datadoghq.com/blog/datadog-datastore/"
+  tag: "Blog"
+  text: "Enhance your automated workflows and apps with Datastore"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">App Builder is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+You can reference and perform CRUD (Create, Read, Update, and Delete) operations on a datastore inside a workflow or an app. Additionally, you can create a workflow or app directly from an existing datastore.
+
+## Create a workflow or app from a datastore {#create-workflow-app}
+
+You can get started with a workflow or app directly from a datastore.
+
+### Workflow Automation
+
+To create a workflow from a datastore:
+1. On the [Datastore page][1], locate your datastore in the list and click to open it.
+1. Click **Create** > **Workflow from Datastore**.
+
+Datadog creates a workflow with a **List items** workflow step prepopulated with your datastore ID. From here, follow the [Workflow Automation][2] documentation to build your workflow. For a list of available datastore actions, see the [Action Catalog][4].
+
+### App Builder
+
+To create an app from a datastore:
+1. On the [Datastore page][1], locate your datastore in the list and click to open it.
+1. Click **Create** > **App from Datastore**.
+
+Datadog creates an app prepopulated with your datastore ID. From here, follow the [App Builder][3] documentation to build your app. For a list of available datastore actions, see the [Action Catalog][4].
+
+## Use a datastore in an app {#use-app}
+
+To use a datastore in an existing app, add a datastore action:
+1. Click the Data (**{&nbsp;}**) icon to open the Data tab.
+1. Click the plus (**+**) icon, select **Action**, and add a Datastore action to add to your app. For a list of available datastore actions, see the [Action Catalog][4].
+1. Choose an existing connection or create one.
```

Comment:
It could be worth linking the connections documentation here, i.e.:

```suggestion
1. Choose an existing connection or [create one](https://docs.datadoghq.com/actions/connections/?tab=workflowautomation#create-a-connection).
```

Connections are notoriously confusing to setup.